examples/oneview_user.yml

```diff
@@ -56,27 +73,144 @@
+        state: validate_user_name
+        data:
+          userName: "testUser"
       delegate_to: localhost
-      register: deleted
 
-    - name: Do nothing when User is absent
+    - debug: var=user
+
+    - name: Checks for the existence of a user with the specified full name 
+      oneview_user:
+        config: "{{ config }}"
+        state: validate_full_name
+        data:
+          fullName: "testUser101"
+      delegate_to: localhost
+
+    - debug: var=user
+
+    - name: Add role to existing username
+      oneview_user:
+        config: "{{ config }}"
+        state: add_role_to_username
+        data:
+          userName: "testUser"
+          role_list:
+            - roleName: "Backup administrator"
+            - roleName: "Scope administrator"
+      delegate_to: localhost
+
+    - debug: var=user
+
+    - name: Update role to existing username
+      oneview_user:
+        config: "{{ config }}"
+        state: update_role_to_username
+        data:
+          userName: "testUser"
+          role_list:
+            - roleName: "Infrastructure administrator"
+            - roleName: "Read only"
+            - roleName: "Scope operator"
+      delegate_to: localhost
+
+    - debug: var=user
+
+    - name: remove role from existing username
+      oneview_user:
+        config: "{{ config }}"
+        state: remove_role_from_username
+        data:
+          userName: "testUser"
+          role_list: 
+            - "Read only"
+            - "Scope operator"
+      delegate_to: localhost
+
+    - debug: var=user
+
+    - name: Delete single user
       oneview_user:
         config: "{{ config }}"
         state: absent
-        data: "{{ user_1.ansible_facts.user }}"
+        data:
+          userName: "testUser"
+      delegate_to: localhost 
+
+    - debug: var=user
+
+
+    - name: Delete multiple users
+      oneview_user:
+        config: "{{ config }}"
+        state: delete_multiple_users
+        data:
+          users_list:
+            - "testUser1"
+            - "testUser2"
       delegate_to: localhost
-      register: deleted
+
+    - debug: var=user
+
+    - name: Create a User for automation
+      oneview_user:
+        config: "{{ config }}"
+        state: present
+        data:
+          userName: "{{ user_name }}"
+          password: "myPass1234"
+          emailAddress: "{{ email }}"
+          enabled: true
+          fullName: "testUser101"
+          mobilePhone: '555-2121'
+          officePhone: '555-1212'
+          permissions:
+            - roleName: "Read only"
+            - roleName: "Infrastructure administrator"
+      delegate_to: localhost
+      register: user_1
+
+#    - name: Sets a new password for the user
+#      oneview_user:
+#        config: "{{ config }}"
+#        state: set_password
+#        data:
+#          userName: "{{ user_name }}"
+#          newPassword: "myNEWPass1234"
+#          oldPassword: "myOLDPass1234"
+#      delegate_to: localhost
```

Comment:
any reason why this has been commented out?

Reply:
This tasks will be only executed during the initial setup of the appliance....(to change password)....